control-plane/pkg/apis/messaging/v1beta1/kafka_channel_lifecycle.go

```diff
@@ -106,11 +98,6 @@ func (cs *KafkaChannelStatus) GetCondition(t apis.ConditionType) *apis.Condition
 	return cs.GetConditionSet().Manage(cs).GetCondition(t)
 }
 
-// IsReady returns true if the resource is ready overall.
-func (cs *KafkaChannelStatus) IsReady() bool {
-	return cs.GetConditionSet().Manage(cs).IsHappy()
-}
-
```

Comment:
In Eventing we have also removed these, in FAV of the above `func (kc *KafkaChannel) IsReady() bool` 

(see IMC on eventing)